internal/driver/driver.go

```diff
@@ -122,6 +133,25 @@ func (d *Driver) Initialize(sdk interfaces.DeviceServiceSDK) error {
 		d.lc.Errorf("failed to register secret update callback: %v", err)
 	}
 
+	// check to see if rtsp-simple-server file/binary exists
+	_, err = os.Stat(RtspServerCmd)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return fmt.Errorf("%s file cannot be found: %s", RtspServerCmd, err.Error())
+		} else if os.IsPermission(err) {
+			return fmt.Errorf("permission denied for %s: %s", RtspServerCmd, err.Error())
+		} else {
+			return fmt.Errorf("unknown error occurred while checking for rtsp-server binary file %s: %s", RtspServerCmd, err.Error())
+		}
+	}
+
+	rtspProc := exec.Command(RtspServerCmd)
+	rtspProc.Stdout = os.Stdout
+	err = rtspProc.Start()
+	if err != nil {
+		return fmt.Errorf("unable to start %s process: %s", RtspServerCmd, err.Error())
+	}
```

Comment:
Isn't the process already started in [docker-entrypoint](https://github.com/edgexfoundry/device-usb-camera/blob/01aee5a5c41b4c067632ab994dacfc003f161152/docker-entrypoint.sh#L4)? 
The snap uses systemctl to start it per app definition here: https://github.com/edgexfoundry/device-usb-camera/blob/01aee5a5c41b4c067632ab994dacfc003f161152/snap/snapcraft.yaml#L35-L39

Reply:
yes, I seem to have forgotten to remove it from the entrypoint script, I will put in a PR to fix this

Reply:
Being addressed in https://github.com/edgexfoundry/device-usb-camera/pull/270